modules/era/sql/item_mods.sql

```diff
@@ -18,7 +18,8 @@ REPLACE INTO `item_mods` (`itemid`, `modid`, `value`) VALUES
     (13887,521,10), -- Black Sallet AUGMENTS_ABSORB: 10
     (14010,521,10), -- Black Gadlings AUGMENTS_ABSORB: 10
     (15400,521,10), -- Black Cuisses AUGMENTS_ABSORB: 10
-    (15339,521,10); -- Black Sollerets AUGMENTS_ABSORB: 10
+    (15339,521,10), -- Black Sollerets AUGMENTS_ABSORB: 10
+    (15365,362,10); -- Drachen Greaves +1 JUMP_ATT_BONUS: 10
```

Comment:
You have no proof drachen greaves +1 was different in era.

Reply:
I apologize, I didn't see your Bluegartr thread. Given it is 2012, I don't really see a reason it was increased. I was simply going off of the wiki (https://ffxiclopedia.fandom.com/wiki/Drachen_Greaves_%2B1?oldid=946685).